allensdk/brain_observatory/behavior/session_metrics.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from allensdk.brain_observatory.behavior import trial_masks as masks
+
+def response_bias(trials, detect_col, trial_types=("go", "catch")):
+    """
+    Calculate the response bias for a subset of trial types from a behavioral
+    training dataframe.
+    Args:
+        trials (pandas.DataFrame): Dataframe containing trial-level information
+            from a behavioral training session. Required columns:
+            "trial_type", `detect_col`.
+        detect_col (str): Name of column containing boolean 
+            or numeric codings (0/1) for whether or not the mouse had a 
+            response.
+        trial_types (iterable<str>): Iterable containing string trial types
+            to check for the response bias. Trials of types not included in this
+            iterable will be ignored. Default=("go", "catch")
+    Return:
+        The response bias (or average value of the `detect_col`)
+        for trials in `trial_types`.
+    """
+    mask = masks.trial_types(trials, trial_types)
+    return trials[mask][detect_col].mean()
+
+
+def num_contingent_trials(session_trials):
```

Comment:
in masks you have a contingent_trials function which (insofar as it is justified) is useful for offering a 1-stop definition of a contingent trial, why not use that here?

Reply:
Agree it's silly, was just moved over from visual_behavior_analysis to not have to change more code than necessary in mtrain_api to remove visual_behavior_analysis as a dependency